pom.xml

```diff
@@ -147,6 +147,11 @@
                 <artifactId>snappy-java</artifactId>
                 <version>${snappy.java.version}</version>
             </dependency>
+            <dependency>
```

Comment:
Can you check from where zookeeper dependency is coming from? Directly adding a new dependency could be avoided

Reply:
@tarunjain-confluent , zookeeper is a dependency of `hive-exec`. The latest stable version of hive-exec pulls in `org.apache.zookeeper:zookeeper:3.4.6` which is also affected by this CVE.

Reply:
https://mvnrepository.com/artifact/org.apache.hive/hive-exec/3.1.3